### PR TITLE
Fixing issues with latest changes to docker autoconfig

### PR DESCRIPTION
**Pull request by nlopezgi**
Fixing release script
fixing rule when multiple external repos are pulled from the container

### CODE REVIEW

File: release/cc/execute_targets.py [hunk 1]
```diff
@@ -83,16 +83,22 @@ def execute_and_extract_configs(container_configs_list, bazel_version):
...
 
-      command = ("bazel run --define=DOCKER_AUTOCONF_OUTPUT={OUTPUT_DIR} "
-                 "//{PACKAGE}:{TARGET}").format(
+      command = ("bazel build //{PACKAGE}:{TARGET}").format(
+                     PACKAGE=container_configs.package,
+                     TARGET=target)
+      print("\nExecuting command: %s\n" % command)
+      subprocess.check_call(shlex.split(command))
+
+      command = ("cp ./bazel-out/k8-fastbuild/bin/{PACKAGE}/{TARGET}_outputs.tar "
```
Comment (xingao267) [inline release/cc/execute_targets.py]: Will this make the tarball left in the {OUTPUT_DIR}, if so, we should delete it after extraction.

File: tests/config/BUILD [hunk 2]
```diff
@@ -108,6 +108,19 @@ docker_toolchain_autoconfig(
     test = True,
 )
 
+docker_toolchain_autoconfig(
```
Comment (xingao267) [inline tests/config/BUILD]: Can we add the test for this to presubmit?

**Comment by xingao267** [comment]
Let's also include this PR in the coming release (release for bazel 0.18.0) as it fixes autoconfig.

File: tests/config/BUILD (hunk 2 above)
Comment (nlopezgi) [reply tests/config/BUILD]: done

**Comment by nlopezgi** [comment]
added new test.
+1 to waiting for this PR to push the new release

File: release/cc/execute_targets.py (hunk 1 above)
Comment (nlopezgi) [reply release/cc/execute_targets.py]: the existing functionality was to output the tar to the OUTPUT_DIR
I'm just keeping it on par by copying it to OUTPUT_DIR after bazel build
 (i.e., this should already be cleaned up by the existing clean up process)

**Pull request merged**